### PR TITLE
add reservoir project management support to cwms-python

### PR DESCRIPTION
**Pull request by adamkorynta**

### CODE REVIEW

**Comment by Enovotny** [comment]
Can you address the 2 sonarcloud issues.  Also I don't want to commit this until the endpoints are in CWBI-test at least.  right now I don't see some of these endpoints in CWBI test yet.

**Comment by adamkorynta** [comment]
Thanks for catching the 2 issues, my eyes glazed over it because of the green checkmark.

Sounds good about holding off on merge.

File: cwms/__init__.py
Comment (Enovotny) [inline cwms/__init__.py:10]: since you made this pull request the _init file structure changed.  please update to cwms.projects....  This is for compatibility with airflow. after that I will approve.

**Review by Enovotny — CHANGES_REQUESTED** [review]

File: cwms/__init__.py
Comment (adamkorynta) [reply cwms/__init__.py]: I rebased and updated the _init

Comment (Enovotny) [reply cwms/__init__.py]: can you add project_locks and project lock rights as well.  looks like you just have projects on the init now.

Comment (adamkorynta) [reply cwms/__init__.py]: added

**Review by Enovotny — APPROVED** [review]

**Pull request merged**
